modules/c++/math.linear/include/math/linear/Matrix2D.h

```diff
@@ -1082,6 +1082,21 @@ class Matrix2D
     }
 
     /*!
+     *  Negation operator;
+     *
+     *  \code
+           B = -A;
+     *  \endcode
+     *
+     */
+    Matrix2D operator-() const
+    {
+        Matrix2D mneg(*this);
+        std::transform(mneg.mRaw, mneg.mRaw+mneg.mMN, mneg.mRaw, std::negate<_T>());
```

Comment:
80 character line limit... also put spaces between operators.  So this should become something like...

```
std::transform(neg.mRaw,
               neg.mRaw + neg.mMN,
               neg.mRaw,
               std::negate<_T>());
```

or

```
std::transform(neg.mRaw, neg.mRaw + neg.mMN, neg.mRaw,
               std::negate<_T>());
```

Personally I like the first one best.